chapters/classes.tex

```diff
@@ -1014,31 +1007,30 @@ \section{Balanced Models}\doublelabel{balanced-models}
   equations that will be provided when the class is used.}{]}.
 \end{itemize}
 
-{[}\emph{To clarify top-level inputs without binding equation (for
+\begin{nonnormative}
+To clarify top-level inputs without binding equation (for
 non-inherited inputs binding equation is identical to declaration
 equation, but binding equations also include the case where another
-model extends M and has a modifier on `u' giving the value):}
-
+model extends \lstinline!M! and has a modifier on \lstinline!u! giving the value):
 \begin{lstlisting}[language=modelica]
 model M
   input Real u;
   input Real u2=2;
 end M;
 \end{lstlisting}
 
-\emph{Here `u' and `u2' are top-level inputs and not connectors. The
-variable u2 has a binding equation, but u does not have a binding
-equation. In the equation count, it is assumed that an equation for u is
-supplied when using the model.}
-
-{]}
+Here \lstinline!u! and \lstinline!u2! are top-level inputs and not connectors. The
+variable \lstinline!u2! has a binding equation, but \lstinline!u! does not have a binding
+equation. In the equation count, it is assumed that an equation for \lstinline!u! is
+supplied when using the model.
+\end{nonnormative}
 
 \textbf{Definition 3: Locally Balanced}
 
-A model or block class is ``locally balanced'' if the ``local number of
-unknowns'' is identical to the ``local equation size'' for all legal
+A model or block class is \firstuse{locally balanced} if the \firstuse{local number of
```

Comment:
Note that this isn't first use of "local number of unknowns", as it was just defined in Definition 1 (which really should be a proper definition-environment, but one thing at a time). I don't know exactly how to improve that.

Reply:
I just moved the use of `\firstuse` to where the concepts were introduced, and switched to '\emph' here.